modules/standard/Types.chpl

```diff
@@ -413,8 +413,8 @@ pragma "no doc"
 proc isProperSubtype(type sub, type super) param
   return false;
 
-
-
+/* Returns `true` if `x` contains a `these()` method. */
+proc isIterable(x) param return canResolveMethod(x, 'these');
```

Comment:
Should this check that ``these`` is an ``iter``, not a ``proc``?

Reply:
I think that's a good idea, but it will require some modifications to the Reflection module and compiler.